dev/field_printer.h

```diff
@@ -85,7 +85,14 @@ namespace sqlite_orm {
             return "null";
         }
     };
-
+#ifdef SQLITE_ORM_OPTIONAL_SUPPORTED
+    template<>
+    struct field_printer<std::nullopt_t, void> {
+        std::string operator()(const std::nullopt_t&) const {
+            return "null";
```

Comment:
You can now use this specialization for null values in `field_printer<std::optional<T>>` specialization instead of `std::nullptr_t`.